### DISCUSSION TITLE
Questions on Form Fast

### DISCUSSION THREAD

**Discussion by osseonews** [alibey-10/indie-ui#55] [Q&A]
First of all, congrats, Form Fast is incredible. A few questions:

- What does it mean in the form when you say: "Select type for ToggleGroup" what is this and how does it work?
- Can you show a sample of using an async onsubmit with errors? Usually the onsubmit is going to be async as you need to call an external service of some sort
- Any examples using dependent fields? Show one field in the form based on the value of another field?

**Comment by alibey-10** [comment]
ToggleGroup is a Shadcn component that renders a list of Toggle components, it can work like Checkbox groups or Radio groups.

Showing success/error messages after submission and Dependent fields aren't supported yet, FormFast is a few weeks old.

Feel free to create new posts to vote on proposed features, I would love to see what the community cares about most. 
@osseonews
